src/words/TokTok.jl

```diff
@@ -54,14 +54,14 @@ const CURRENCY_SYM = (
         "\u20b4", "\u20b5", "\u20b6", "\u20b7", "\u20b8", "\u20b9", "\u20ba", "\ua838",
         "\ufdfc", "\ufe69", "\uff04", "\uffe0", "\uffe1", "\uffe5", "\uffe6", "\u09fb")
 
-const rules_atoms = Tuple(Iterators.flatten([
+const rules_atoms = collect.(Tuple(Iterators.flatten([
```

Comment:
The `String` error occurring in `atoms` was due to this. I tested it on the wiki corpus that @ComputerMaestro was getting error in, and now this works.

Reply:
I have updated the testset for this as well.